module/documents/item.mjs

```diff
@@ -2164,6 +2164,9 @@ export default class Item5e extends SystemDocumentMixin(Item) {
     // Get spell data
     const itemData = (spell instanceof Item5e) ? spell.toObject() : spell;
 
+    // improve scroll creation so that effects and flags from the original spell are included in the created scroll
+    options = mergeObject(options, {flags: itemData.flags, effects: itemData.effects}, {overwrite: false});
```

Comment:
This seems better handled down on line 2218.

Reply:
Are you sure ?  I mean you're Zhell so you're probably right, but in line 2218 I only see property management for  'name', 'image' and 'system' no ?

Reply:
I agree with Zhell that this is better handled in the section of code where we create the spell scroll data, since we are already including information from the item data there. The item data can still be modified in the `preCreateScrollFromSpell` hook, leading to the same outcome as if they had modified `options` instead.